components/admin/requests/reports/GenerateModal.tsx

```diff
@@ -79,6 +79,13 @@ export default function GenerateReportModal(props: Props) {
   >(GENERATE_APPLICATIONS_REPORT_QUERY, {
     onCompleted: data => {
       if (data.generateApplicationsReport.ok) {
```

Comment:
```suggestion
      if (data.generateApplicationsReport.ok && !!data.generateApplicationsReport.link) {
```
and after we change the param to `url`, update this accordingly. This should allow us to remove the string type declaration in the `link.setAttribute` call